static/src/js/components/SearchPanels/SearchPanels.js

```diff
@@ -280,6 +280,8 @@ class SearchPanels extends PureComponent {
     ]
 
     const setCollectionSort = (value) => {
+      if (value === activeCollectionsSortKey) return
```

Comment:
Can you add a test for this in `static/src/js/components/SearchPanels/__tests__/SearchPanels.test.js`

Reply:
Hey so after looking over this extensively, I am unsure how to get the children components to render in such a way that I may `simulate('mouseOver')` and show the dropdown then `simulate('click')` and assert the functions do or do not call. 

The showing and clicking of the sort drop down seems to be taken care of here, 

https://github.com/nasa/earthdata-search/blob/584ba390b60b828c0895c331557ded14cc9ad899/static/src/js/components/Panels/__tests__/PanelGroupHeader.test.js#L158

What do you think I should/can do?